packages/core/utils/src/modules-sdk/migration-scripts/migration-down.ts

```diff
@@ -58,7 +58,7 @@ export function buildRevertMigrationScript({
       logger?.error(
         `${upperCaseFirst(
           moduleName
-        )} module migration failed to run - Error: ${error}`
+        )} module migration failed to run - Error: ${error.errros ?? error}`
```

Comment:
question: Is this a typo or is it really`errros`? 😄

Reply:
This is returning an `AggregateError` for some errors. I suspect there is a Promise.reject + new Error somewhere (not in our packages), like connection refused.
So this should solve and display the message at least.